### PR TITLE
Set model in eval mode when inferencing.

### PR DESCRIPTION
**Pull request by kylegao91**
Address #33

### CODE REVIEW

File: seq2seq/evaluator/evaluator.py [hunk 1]
```diff
@@ -48,4 +48,6 @@ def evaluate(self, model, data):
                 # Evaluate loss
                 loss.eval_batch(batch_output, batch_target)
 
+        model.train(True)
```
Comment (kpfadnis) [inline seq2seq/evaluator/evaluator.py]: @kylegao91 since evaluator should never evaluate with dropout, I would recommend not setting `model.train(True)` in here. Let the trainer handle it. 

I prefer only setting `model.eval()` in the evaluator.

Comment (kylegao91) [inline seq2seq/evaluator/evaluator.py]: @kpfadnis Makes sense.  Pushed changes for this.
